pkgs/by-name/za/zarf/package.nix

```
@@ -6,38 +6,66 @@
   stdenv,
```

```suggestion
  stdenv,
  git,
```
Fixes the CI failure